interface/main/messages/messages.php

```diff
@@ -1,668 +1,885 @@
 <?php
```

Comment:
TODO
Look through commit history of this script to ensure don't lose any features/changes that were recently added (such as several things involving patient messaging)